lib/broadway/server.ex

```diff
@@ -30,7 +34,8 @@ defmodule Broadway.Server do
        supervisor_pid: supervisor_pid,
        terminator: config.terminator,
        name: opts[:name],
-       producers_names: producer_names(name_prefix(opts[:name]), config.producer_config)
+       producers_names: producer_names(name_prefix(opts[:name]), config.producer_config),
+       rate_limiting?: not is_nil(config.producer_config[:rate_limiting])
```

Comment:
Have it store the rate limiting actual name or nil otherwise.

Reply:
Done ✅